.vscode/launch.json

```diff
@@ -13,6 +13,12 @@
                 "/var/www/html/": "${workspaceFolder}/docker/wordpress",
                 "/var/www/html/wp-content/plugins/woocommerce-gateway-stripe/": "${workspaceFolder}"
             }
+        },
+        {
+            "name": "Xdebug local",
```

Comment:
It has absolutely nothing to do with the PR... but maybe we can name those two something like:
- `Listen for Xdebug (remote)`
- `Listen for Xdebug (local)`

Reply:
Yeah, this was not supposed to be included 🤦‍♂️ 
I'll remove this. I need to open a separate PR for that.

Reply:
Removed in 94b2851de80540adc1f34f4ac966747cd7a914eb

Reply:
I meant that my request had nothing to do with the PR... I like the idea of having both Xdebug configs, I did not like the naming though 🤣 . But we can add them in another PR if you prefer..

Reply:
Haha I see what you mean! I think it should still be part of a separate PR 😛